src/config.ts

```diff
@@ -16,6 +16,15 @@ export const MARKETPLACE_SALEOR_APPS_JSON_PATH =
 export const MARKETPLACE_APP_TEMPLATE_GALLERY_PATH =
   window.__SALEOR_CONFIG__.APP_TEMPLATE_GALLERY_PATH;
 
+export const AppsConfig = {
+  marketplaceUrl:
+    window.__SALEOR_CONFIG__.APPS_MARKETPLACE_URL ||
+    "https://marketplace-gray.vercel.app/api/v2/saleor-apps",
```

Comment:
Please do not hardcode this url. It should be *optional* and provided only via env. If it doesn't exist, Marketplace should be not rendered

Reply:
Sure, fixed